community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/cache/OffHeapNumberArray.java

```diff
@@ -52,7 +52,11 @@ public void close()
     {
         if ( !closed )
         {
-            UnsafeUtil.free( address );
+            if ( length > 0 )
```

Comment:
not sure we can test for it. Actually this may simply be an issue with the added tracking of allocation/access which can be enabled by a JVM flag, `CHECK_NATIVE_ACCESS`. Since UnsafeUtil can "allocate" 0 then deallocating 0 should be OK as well, sort of.